src/com/example/ui.cljs

```diff
@@ -77,7 +80,12 @@
            {:quiz/questions (comp/get-query Question)}
            fs/form-config-join]
    :ident (fn [] [:quiz/id id])
-   :initial-state {}}
+   :initial-state {}
+   :route-segment ["quiz" :quiz-id]
+   :will-enter (fn [app {:keys [quiz-id]}]
+                 (dr/route-deferred [:quiz/id quiz-id]
+                                    #(df/load! app [:quiz/id quiz-id] Quiz {:post-mutation `dr/target-ready
+                                                                            :post-mutation-params {:target [:quiz/id quiz-id]}})))}
```

Comment:
instead of loading the quiz data when the whole app loads, we load the quiz data as the user navigates to this route.

`route-deferred` tells fulcro not to show anything on the screen until `target-ready` is called. This lets us wait for the data to load and then display the screen.